guides/common/modules/con_provisioning-virtual-machines-on-kubevirt.adoc

```diff
@@ -1,32 +1,20 @@
 [id="Provisioning_Virtual_Machines_Kubevirt_{context}"]
 = Provisioning Virtual Machines on {KubeVirt}
 
-{KubeVirt} addresses the needs of development teams that have adopted or want to adopt Kubernetes but possess existing virtual machine (VM)-based workloads that cannot be easily containerized.
+{KubeVirt} addresses the needs of development teams that have adopted or want to adopt {Kubernetes} but possess existing virtual machine (VM) workloads that cannot be easily containerized.
 This technology provides a unified development platform where developers can build, modify, and deploy applications residing in application containers and VMs in a shared environment.
 These capabilities support rapid application modernization across the open hybrid cloud.
 
-With {Project}, you can create a compute resource for {KubeVirt} so that you can provision and manage Kubernetes virtual machines using {Project}.
+With {Project}, you can create a compute resource for {KubeVirt} so that you can provision and manage virtual machines in {Kubernetes} using {Project}.
 
 ifdef::satellite[]
 Note that template provisioning is not supported for this release.
```

Comment:
@ekohl I'm not sure what this line about template provisioning means. It was added 4 years ago. Do you know whether it is valid?

I removed the Tech Preview note because it is probably GA by now.

Reply:
Adding @ares

Reply:
I don't know Kubevirt, but I'm guessing that it means using templates. Whether that's templates within Foreman or in Kubevirt is an excellent question I don't know the answer to.